lib/src/floodFilling.js

```diff
@@ -0,0 +1,352 @@
+/* eslint-disable */
```

Comment:
remove

Reply:
This would be in the FloodFilling PR

Reply:
Ah, then remove from this PR, using ```git restore --staged <filename>``` for any files that are for the other PR